GliaWidgets/Sources/View/Chat/Message/GvaResponseTextView.swift

```diff
@@ -0,0 +1,84 @@
+import UIKit
+
+class GvaResponseTextView: ChatMessageView {
+    var showsOperatorImage: Bool = false {
+        didSet {
+            if showsOperatorImage {
+                guard operatorImageView == nil else { return }
+                let operatorImageView = UserImageView(
+                    with: viewStyle.operatorImage,
+                    environment: .init(
+                        data: environment.data,
+                        uuid: environment.uuid,
+                        gcd: environment.gcd,
+                        imageViewCache: environment.imageViewCache
+                    )
+                )
+                self.operatorImageView = operatorImageView
+                operatorImageViewContainer.addSubview(operatorImageView)
+                operatorImageView.layoutInSuperview().activate()
+            } else {
+                operatorImageView?.removeFromSuperview()
+                operatorImageView = nil
+            }
+        }
+    }
+
+    private let viewStyle: OperatorChatMessageStyle
+    private var operatorImageView: UserImageView?
+    private var operatorImageViewContainer = UIView().makeView()
```

Comment:
Could it be `let`?

Reply:
fixed [114977d](https://github.com/salemove/ios-sdk-widgets/pull/673/commits/114977de071802ccf684ae14daa3ea299a7b842c)